src/Exception/ExceptionInterface.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Bernard\Exception;
+
+/**
+ * Interface for exceptions thrown by Bernard
+ * @package Bernard
+ */
+interface ExceptionInterface
```

Comment:
I think that it would be an edge case to catch all exceptions from Bernard instead of the specific ones, so i would vote for just `Exception` unless it is a protected keyword.